### PR TITLE
Update cfp.html

### PR DESCRIPTION
**Pull request by ramonh**
Add some changes as suggested here:
https://twitter.com/benschwarz/status/983564931210428416

### CODE REVIEW

File: cfp.html [hunk 1]
```diff
@@ -16,7 +16,11 @@ <h1 class="c-red">
             <p>You’ll be joining over 16 years of tradition in inspiring and amazing Rubyists and other developers around Europe.</p>
             <p>A few things to note:</p>
             <ul>
+                <li>EuRuKo is a not-for-profit event.</li>
```
Comment (giedriusr) [inline cfp.html]: I think `non-profit` is more common expression to describe such type.

Comment (ramonh) [reply cfp.html]: Oh wonderful, thank you!

File: cfp.html [hunk 2]
```diff
@@ -16,7 +16,11 @@ <h1 class="c-red">
...
                 <li>All talks will have to adhere to our <a href="/coc">Code of Conduct</a>.</li>
+                <li>We are fully committed to have a lineup as diverse as we can.</li>
+                <li>As a not-for-profit event, we cannot offer to pay you for speaking. We can, however, cover travel and accommodation (see the perks section).</li>
+                <li>We will dedicate a few slots speakers that we will especially invite, including Matz.</li>
```
Comment (giedriusr) [inline cfp.html]: How about: `We will dedicate a few slots for special speakers such as Matz.`

Comment (ramonh) [reply cfp.html]: That's much better, thank you!

Comment (giedriusr) [reply cfp.html]: Not sure if a is needed before few, I think we don't need it. `We will dedicate few slots`

File: cfp.html [hunk 3]
```diff
@@ -36,10 +40,9 @@ <h1 style="margin-top: 50px" class="section-header align-self-start">Speaker ass
             <p>If you’re a first time speaker and thinking EuRuKo is out of your reach, you’re mistaken! We encourage first-time speakers to use EuRuKo as a springboard for a career of awesome talks!</p>
             <p>Ruby’s in the name, so Ruby should be the game! Rails talks are cool, of course, but the content of the conference is framework-agnostic. And hey, if you’ve got a non-Ruby or ‘for Rubyists’ talk in mind, send it our way!</p>
             <h1 style="margin-top: 50px" class="section-header align-self-start">The perks</h1>
-            <p>What do you get as a speaker, I hear you 🤔’ing? If you’re selected as a speaker, here’s what we can offer you:</p>
+            <p>What do you get as a speaker, I hear you 🤔’ing? Aside from having free entry to the conference (which really barely counts as a perk!), if you’re selected as a speaker, here’s what we can offer you:</p>
```
Comment (giedriusr) [inline cfp.html]: Not sure if this emoji is descriptive enough, at least when I first read, I had to spend extra second to understand the word. Maybe simple text is better? Not sure how does this reflect on the website. What others think?

Comment (ramonh) [reply cfp.html]: You're right. I was trying to be cute, but I think it's a bit obscure. Thank you for the feedback. I'll change it

File: cfp.html (hunk 2 above)
Comment (ramonh) [reply cfp.html]: Hmm, I think that changes the meaning.

How do you feel about `... a couple...`?

File: cfp.html [hunk 4]
```diff
@@ -16,7 +16,11 @@ <h1 class="c-red">
             <p>You’ll be joining over 16 years of tradition in inspiring and amazing Rubyists and other developers around Europe.</p>
             <p>A few things to note:</p>
             <ul>
+                <li>EuRuKo is a non-profit event.</li>
                 <li>All talks will have to adhere to our <a href="/coc">Code of Conduct</a>.</li>
+                <li>We are fully committed to have a lineup as diverse as we can.</li>
+                <li>As a non-profit event, we cannot offer to pay you for speaking. We can, however, cover travel and accommodation (see the perks section).</li>
+                <li>We will dedicate a few slots for special speakers such as Matz.</li>
```
Comment (vindia) [inline cfp.html]: "We will dedicate a few slots for invited speakers, like Matz."

And maybe add to that "Invited speakers are not paid a fee either, but do not have to go through the CfP process." (if true :grin:)

Comment (ramonh) [reply cfp.html]: Which it is, yes! Thanks!

File: cfp.html (hunk 2 above)
Comment (giedriusr) [reply cfp.html]: Sounds right to me when singularity is used, for instance: `A slot will be dedicated for X.` versus `Few slots will be dedicated for X.` But I might be wrong since I am not a native speaker :) Let's leave it as it is.

File: cfp.html [hunk 5]
```diff
@@ -36,10 +40,9 @@ <h1 style="margin-top: 50px" class="section-header align-self-start">Speaker ass
...
             <p>Ruby’s in the name, so Ruby should be the game! Rails talks are cool, of course, but the content of the conference is framework-agnostic. And hey, if you’ve got a non-Ruby or ‘for Rubyists’ talk in mind, send it our way!</p>
             <h1 style="margin-top: 50px" class="section-header align-self-start">The perks</h1>
-            <p>What do you get as a speaker, I hear you 🤔’ing? If you’re selected as a speaker, here’s what we can offer you:</p>
+            <p>What do you get as a speaker, I hear you asking? Aside from having free entry to the conference (which really barely counts as a perk!), if you’re selected as a speaker, here’s what we can offer you:</p>
             <ul>
-                <li>A free ticket to the conference</li>
-                <li>If your employer can cover your travel and accommodation, that’d be amazing! You could even show them on a slide!</li>
+                <li>We will cover travel and accommodation for you. If your employer can cover your travel and accommodation though, that’d be amazing! You could even show them on a slide!</li>
```
Comment (vindia) [inline cfp.html]: > "You could even show them on a slide!" 

That sounds a bit weird to me. Probably all speakers will have a slide saying something like "Hey, my name is FOO and I work at BAR", right?

I would rephrase it like this (feedback please!)

> We will cover travel and accommodation for you. However, since we're a non-profit event, if your employer offers a budget for travel and accommodation we kindly ask you to use that. This way we can use our budget for the speakers that don't have these benefits.

File: cfp.html [hunk 6]
```diff
@@ -16,7 +16,11 @@ <h1 class="c-red">
             <p>You’ll be joining over 16 years of tradition in inspiring and amazing Rubyists and other developers around Europe.</p>
             <p>A few things to note:</p>
             <ul>
+                <li>EuRuKo is a non-profit event.</li>
                 <li>All talks will have to adhere to our <a href="/coc">Code of Conduct</a>.</li>
+                <li>We are fully committed to have a lineup as diverse as we can.</li>
```
Comment (vindia) [inline cfp.html]: Maybe here link to the process we're doing in the CFP (or just the text "See our CFP process guidelines for more info")

File: cfp.html [hunk 7]
```diff
@@ -36,10 +40,9 @@ <h1 style="margin-top: 50px" class="section-header align-self-start">Speaker ass
...
             <h1 style="margin-top: 50px" class="section-header align-self-start">The perks</h1>
-            <p>What do you get as a speaker, I hear you 🤔’ing? If you’re selected as a speaker, here’s what we can offer you:</p>
+            <p>What do you get as a speaker, I hear you asking? Aside from having free entry to the conference (which really barely counts as a perk!), if you’re selected as a speaker, here’s what we can offer you:</p>
             <ul>
-                <li>A free ticket to the conference</li>
-                <li>If your employer can cover your travel and accommodation, that’d be amazing! You could even show them on a slide!</li>
+                <li>We will cover travel and accommodation for you. If your employer can cover your travel and accommodation though, that’d be amazing! You could even show them on a slide!</li>
                 <li>Please let us know how we can help you out otherwise!</li>
             </ul>
             <p>For details and the submission procedure, please take a look at our CFP app:</p>
```
Comment (vindia) [inline cfp.html]: Maybe add "or for any questions regarding visiting the conference as a speaker, email us at ..."

File: cfp.html (hunk 5 above)
Comment (ramonh) [reply cfp.html]: Yep sounds good to me, thank you!

**Comment by vindia** [comment]
👌

**Comment by giedriusr** [comment]
🤞

**Comment by ramonh** [comment]
I made one more addition at the suggestion from @Althaire . Could you please take one more look, please?

**Comment by vindia** [comment]
Nice one, :shipit:

**Comment by ramonh** [comment]
Off it goes, then. Thank you @giedriusr @vindia and @Althaire 🙌
